Gaming The Syllabus/Assets/XML/Enemies/FinalBoss.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
```

Comment:
Gotta say it's really cool to see all our work come together to make this possible